### ISSUE TITLE
systemd-timesyncd just not sync with Windows Server 2012 Domain Controller

### ISSUE THREAD

**Issue by andreybrasil** [systemd/systemd#13912]
**systemd version the issue has been seen with**
> systemd 242 (242)

**Used distribution**
 > Description:    Lubuntu 19.10

**Expected behaviour you didn't see**
 > Time synchronization with Windows Server 2012 Domain Controller with NTP Service enabled. 

**Unexpected behaviour you saw**
 > Time synchronization just doesn't happen the same way it does in Windows Server 2008 R2 Domain Controller or when using ntpd in client.

**Steps to reproduce the problem**
> Just edit /etc/systemd/timesyncd.conf  and fill in the NTP parameter with the address of a Windows Server 2012 Domain Controler with NTP enabled.

>Restart the service and check the logs or run 'timedatectl timesync-status':

>root@mylubuntu:~# timedatectl timesync-status
       Server: X.X.X.11 (ad1-2012.mydomain.com)
Poll interval: 8min 32s (min: 32s; max 34min 8s)
 Packet count: 0

> With 2008 R2:
root@mylubuntu:~# timedatectl timesync-status
       Server: X.X.X.10 (ad1-2008.mydomain.com)
Poll interval: 1min 4s (min: 32s; max 34min 8s)
         Leap: normal
      Version: 3
      Stratum: 2
    Reference: 564D5450
    Precision: 15.625ms (-6)
Root distance: 26.488ms (max: 5s)
       Offset: -28.004ms
        Delay: 1.770ms
       Jitter: 0
 Packet count: 1
    Frequency: +48,508ppm

>WIth ntpdate
root@mylubuntu:~# ntpdate ad1-2012.mydomain.com
1 Nov 09:31:38 ntpdate[13941]: adjust time server  X.X.X.11 offset 0.008825 sec

**Comment by yuwata** [comment]
Please enable debug logging of `systemd-timesyncd`, and provide the log. The debugging log can be generated by writing the following lines in `/etc/systemd/system/systemd-timesyncd.service.d/override.conf`:
```
[Service]
Environment=SYSTEMD_LOG_LEVEL=debug
```

**Comment by andreybrasil** [comment]
Hello,

I enabled debug as requested and identified the problem. Among several lines that previously did not appear due to not having debug enabled, I found this line:

systemd-timesyncd[4199]: Server has too large root distance. Disconnecting.

In my opinion, this kind of error should go to the log, regardless of having debug enabled. I think this is the bug.

During my tests, when it syncs, it creates a record. When it does not synchronize for an error, it does not create any log records unless the server cannot be found.

Thank you.
